config.lua.dist

```diff
@@ -6,6 +6,9 @@ useAnyDatapackFolder = false
 dataPackDirectory = "data-otservbr-global"
 -- Don't change this unless you know what you're doing
 coreDirectory = "data"
+-- Enable/disable development mode
+-- NOTE: In development mode, additional logs will be enabled, for deputation and testing purposes.
+developmentMode = true
```

Comment:
Curious: is this necessary? If we emit logs in at debug or trace wouldn't it suffice to set the log level to `debug`?

Reply:
Debugging can be a challenging task, often involving cumbersome processes and significant time investment to set up the debugging environment. Additionally, testing complex systems can be quite difficult. To address these challenges, I have implemented a comprehensive logging system that allows for easy debugging with the ability to toggle logging on and off.

By incorporating logging into the system, I have provided a way to track and capture relevant information during runtime. This allows for detailed analysis and identification of potential issues or bugs. With logging, you can selectively enable or disable the logging output, providing flexibility in debugging based on our specific needs.

Reply:
I didn't mean that kind of debug. I mean the log level. Spdlog has a set Level method to output trace, debug, info, etc.

Reply:
I see, that makes sense. I am used to log level being configurable outside of the compilation mode to be honest, which is why i though to mention this.